src/iw3unp29.f

```diff
@@ -4569,20 +4554,17 @@ SUBROUTINE S06O29(IDEN,ID)
       RETURN
       END
 
+C> @brief Finds the location of the next numeric character
```

Comment:
If this is a subroutine doxygen block, it should not have @brief, and should not be separated by a blank line from the params. We only need one file-level block, at the top of the file...